out/index.html

```diff
@@ -0,0 +1,87 @@
+</head>
+<body>
+ <div class="section section-0">
+  <div class="container">
+    <a title="To SIS Project Home" href="http://sisproject.berkeley.edu/">
+      <div class="logo"></div>
+    </a>
+  </div>
+</div>
+<div class="section section-1">
+  <div class="container">
+   <div class="square-container">
```

Comment:
@winstonkim indentation still looks a bit off, can you make sure we use the 2 spaces everywhere?